ReadMe.md

```diff
@@ -166,6 +166,40 @@ We are primarily considering JavaScript (sorry, not TypeScript) as the base lang
 
 ## FAQs
 
+#### About OpenSource Projects
+
+ OpenSource Projects means anybody is free to use, study and modify your project for any purpose. The open source community provides a great opportunity for aspiring programmers to distinguish themselves; and by contributing to various projects, developers can improve their skills and get inspiration and support from like-minded people. But most importantly, they can prove that they can build fantastic experiences that people love. means anybody is free to use, study, modify, and distribute your project for any purpose.
+
+
+### How to start Contributing in OpenSource Projects
+
+For doing this we must have to go step by step:-
+* Fork the repository.
+* Clone Your Repository.
+* Copy the link.
+* Go to git bash and write a command $ git clone https://<User Name>@link.
+* Go to your current branch.
+* $ git checkout -b <issue Your are working on>       (This will change the branch from master branch)
+* Make the changes required.
+* After making changes save it and again go to git bash
+* $ git status    (This will show which files You have modified)
+* $ git diff      (This will show you the exact changes in green color)
+* $ git add . (This will add all modifed files)
+* $ git commit -m "Fixed ISSUE
+   >
+   > ISSUE NO."
+* $ git push origin <Current branch name>.
+* Copy the Pull request link.
+* Open the link in your browser.
+* Scroll down to see all  changes you have done.
+* Click on option Create Pull Request.
```

Comment:
Already Available. You can remove this. 😊

Reply:
Okay sir.